### PR TITLE
fix(parser): show more parser context

### PR DESCRIPTION
**Pull request by andylokandy**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

- Closes #issue

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/12898)

### CODE REVIEW

File: src/query/ast/tests/it/testdata/statement-error.txt [hunk 1]
```diff
@@ -531,7 +551,16 @@ error:
...
+  | ----                                ^ expected `)`, `TYPE`, `COMPRESSION`, `FORMAT_NAME`, `RECORD_DELIMITER`, `FIELD_DELIMITER`, `QUOTE`, `NAN_DISPLAY`, `NULL_DISPLAY`, `ESCAPE`, `ROW_TAG`, `SKIP_HEADER`, or `NON_DISPLAY`
+  | |                                   
+  | while parsing `COPY
+                INTO { internalStage | externalStage | externalLocation | [<database_name>.]<table_name> }
+                FROM { internalStage | externalStage | externalLocation | [<database_name>.]<table_name> | ( <query> ) }
+                [ FILE_FORMAT = ( { TYPE = { CSV | JSON | PARQUET } [ formatTypeOptions ] } ) ]
+                [ FILES = ( '<file_name>' [ , '<file_name>' ] [ , ... ] ) ]
+                [ PATTERN = '<regex_pattern>' ]
+                [ VALIDATION_MODE = RETURN_ROWS ]
+                [ copyOptions ]`
```
Comment (andylokandy) [inline src/query/ast/tests/it/testdata/statement-error.txt:563]: now will show example for `copy into statement`

Comment (BohuTANG) [reply src/query/ast/tests/it/testdata/statement-error.txt:563]: Great!

**Review by BohuTANG — APPROVED** [review]

File: src/query/ast/tests/it/testdata/statement-error.txt (hunk 1 above)
Comment (andylokandy) [reply src/query/ast/tests/it/testdata/statement-error.txt:563]: @youngsofun  but the example seems outdated. the [document](https://databend.rs/doc/sql-commands/dml/dml-copy-into-table) is also outdated.

Comment (BohuTANG) [reply src/query/ast/tests/it/testdata/statement-error.txt:563]: The doc updated, remove the `NDJSON`.

**Pull request merged**